src/Symfony/Component/HttpFoundation/Exception/BadRequestException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation\Exception;
+
+/**
+ * Raised when a user sends a malformed request.
+ */
+class BadRequestException extends \UnexpectedValueException implements RequestExceptionInterface
```

Comment:
should we update the exception listener to convert BadRequestException (foundation) into BadRequestHttpException (kernel), that's the intended goal no? 400 by default :)

Reply:
shouldn't this be caught since its an instance of `RequestExceptionInterface` ?

Reply:
correct, my bad.